components/centraldashboard/public/components/main-page.pug

```diff
@@ -12,8 +12,11 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
             a(href$='[[_buildHref("/", queryParams)]]', tabindex='-1')
                 paper-item.menu-item Home
                 aside.divider
```

Comment:
Move divider under the Docs link

Reply:
Sounds good. I also moved it out of the anchor tag since we really don't want that to be clickable. The CSS has been changed slightly to account for that.